app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/registration/RegistrationViewModel.kt

```diff
@@ -27,12 +27,12 @@ class RegistrationViewModel
     }
 
     private fun onRegistrationSuccess(userEntry: UserEntry) {
-        requestFailed.set(false)
+        isRequestSuccess.set(false)
         stateLiveData.value = RegistrationSuccessState()
     }
 
     private fun onError(error: Throwable) {
-        requestFailed.set(true)
+        isRequestSuccess.set(true)
```

Comment:
U set true for `request success` variable inside onError

Reply:
Fck